### DISCUSSION TITLE
When will the Authentication options be included in the templates?

### DISCUSSION THREAD

**Discussion by poepoe12002** [unoplatform/uno.extensions#1330] [General]
Just created a project with the Uno Platform Extension (4.8.8.8) and could not find any of the authentication options.
When will the Authentication options be included in the "Uno Platform App" project templates?

**Comment by nickrandolph** [comment]
Hi @poepoe12002 we've got plans to include samples for this in the next week or so. There'll also be an update to documentation which should make getting started with authentication easier.

**Reply by poepoe12002** [reply]
Great, thanks.
